teps/0091-trusted-resources.md

```diff
@@ -264,22 +260,30 @@ data:
 Example of the Key CRD:
 ```yaml
 apiVersion: tekton.dev/v1alpha1
-Kind: VerificationPolicy
+kind: VerificationPolicy
 metadata:
- name: verification-policy
- namespace: tekton-pipelines
+  name: verification-policy
+  namespace: tekton-pipelines
 spec:
-   resources:
-    - https://github.com/tektoncd/catalog.git
-    - gcr.io/tekton-releases/catalog/upstream/*
-   authorities:
-    - name: cosign
-      key: /etc/signing-secrets/cosign.pub
-    - name: kms
-      key: gcpkms://projects/<project>/locations/<location>/keyRings/<keyring>/cryptoKeys/<key>
-    - name: keyless
-      keyless:
-        url: "https://fulcio.sigstore.dev"
+  resourceAuthorities:
+      # pattern is used to filter out remote resources by their sources URL. e.g. if the resources from https://github.com/tektoncd/catalog.git, then the pattern should be set to https://github.com/tektoncd/catalog.git. ".*" will match any resources.
+    - pattern: ".*"
```

Comment:
Is there an order in which the patterns are applied?
In the above case it would be better if https://github.com/tektoncd/catalog.git is first checked and then ".*".

Reply:
Yes `https://github.com/tektoncd/catalog.git`  is first checked. The current method is that if the pattern is empty, we will treat it as `.*`.